src/artemis/experiment_plans/experiment_registry.py

```diff
@@ -29,12 +35,14 @@ def do_nothing():
         "run": fast_grid_scan_plan.get_plan,
         "internal_param_type": FGSInternalParameters,
         "experiment_param_type": GridScanParams,
+        "callback_collection_type": FGSCallbackCollection,
     },
     "full_grid_scan": {
         "setup": full_grid_scan.create_devices,
         "run": full_grid_scan.get_plan,
         "internal_param_type": FGSInternalParameters,
         "experiment_param_type": GridScanParams,
+        "callback_collection_type": NullPlanCallbackCollection,
     },
     "rotation_scan": {
         "setup": do_nothing,
```

Comment:
Could add `"callback_collection_type": NullPlanCallbackCollection` here as we discussed